docs/kyma/docs/05-02-custom-component-installation.md

```diff
@@ -0,0 +1,110 @@
+    ```
+    - name: "monitoring"
+      namespace: "kyma-system"
+    ```
+  * in the `kyma-installer-local.yaml` file for the **local** installation
+  * in the `kyma-installer-cluster.yaml` file for the **cluster** installation
+
+3. Follow the installation steps to [install Kyma locally from the release](#installation-install-kyma-locally) or [install Kyma on a cluster](#installation-install-kyma-on-a-cluster).
+
+### Installation from sources
+
+1. Customize installation by adding a component to the list of components or removing hashtags (#) in front of the `name` and `namespace` entries in the following installation files:
+  * [`installer-cr.yaml.tpl`](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr.yaml.tpl) for the **local** installation
+  *  [`installer-cr-cluster.yaml.tpl`](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-cluster.yaml.tpl) for the **cluster** installation
+
+2. Follow the installation steps to [install Kyma locally from sources](#installation-install-kyma-locally) or [install Kyma on a cluster](#installation-install-kyma-on-a-cluster).
+
+### Post-installation changes
+
+To add a component that was not installed with Kyma by default, modify the Installation custom resource.
+
+1. Edit the resource:
+    ```
+    kubectl edit installation kyma-installation
+    ```
+2. Add the new component to the list of components or remove hashtags (#) preceding these lines:
+    ```
+    #- name: "jaeger"
+    #  namespace: "kyma-system"
+    ```
+3. Trigger the installation:
+
+   ```
+   kubectl label installation/kyma-installation action=install
+   ```
+
+### Verify the installation
+
+You can verify the installation status by calling `./installation/scripts/is-installed.sh` in the terminal.
+
+## Remove a component
+
+You can only remove the component before the installation process starts. To disable a component from the list of components that you install with Kyma by default, remove this component's `name` and `namespace` entries from the appropriate installation file or add hashtags (#) in front of them. The file differs depending on whether you install Kyma from the release or from sources, and if you install Kyma locally or on a cluster. The version of your component's deployment must match the version that Kyma currently supports.
+
+>**NOTE:** Not all components can be simply removed from the Kyma installation. In case of Istio and the Service Catalog, you must provide your own deployment of these components in the Kyma-supported version before you remove them from the installation process. See [this](https://github.com/kyma-project/kyma/blob/master/resources/istio-kyma-patch/templates/job.yaml#L25) file to check the currently supported version of Istio. See [this](https://github.com/kyma-project/kyma/blob/master/resources/service-catalog/charts/catalog/values.yaml#L3) file to check the currently supported version of the Service Catalog.
```

Comment:
```suggestion
>**NOTE:** For some components, you must perform additional actions to remove them from the Kyma installation. In case of Istio and the Service Catalog, you must provide your own deployment of these components in the Kyma-supported version before you remove them from the installation process. See [this](https://github.com/kyma-project/kyma/blob/master/resources/istio-kyma-patch/templates/job.yaml#L25) file to check the currently supported version of Istio. See [this](https://github.com/kyma-project/kyma/blob/master/resources/service-catalog/charts/catalog/values.yaml#L3) file to check the currently supported version of the Service Catalog.
```